src/clusterfuzz/_internal/tests/appengine/libs/issue_filer_test.py

```diff
@@ -422,6 +477,50 @@ def setUp(self):
     self.mock.get_issue_description.return_value = 'Issue'
     self.mock.utcnow.return_value = datetime.datetime(2016, 1, 1)
 
+  def test_filed_issue_google_issue_tracker(self):
+    """Tests issue filing flow for chromium with google_issue_tracker."""
+
+    helpers.patch(self, [
+        'clusterfuzz._internal.config.local_config.IssueTrackerConfig.get',
+        'clusterfuzz._internal.issue_management.google_issue_tracker.IssueTracker._execute',
+    ])
+    # This mock is for the issue_tracker_utils.get_issue_tracker call.
+    self.mock.get.return_value = CHROMIUM_GIT_POLICY
+    # This mock is for the IssueTrackerConfig.get call.
+    self.mock.get.return_value = {
+        'type': 'google_issue_tracker',
+        'default_component_id': '1234567890',
+        'component_id': '123',
+    }
+    exp_issue_id = 68828938
+    self.mock._execute.return_value = {
+        'issueId': exp_issue_id,
+        'issueState': {
+            'componentId': '29002',
+            'type': 'BUG',
+        },
+    }
+    issue_tracker_utils._ISSUE_TRACKER_CONSTRUCTORS = {
+        'google_issue_tracker': google_issue_tracker.get_issue_tracker
+    }
+    # To avoid calls to real discovery service.
+    client_patcher = mock.patch('clusterfuzz._internal.issue_management.' +
```

Comment:
Does helpers.patch not work for these?

Reply:
I was about to say that I could not get it to work, but tried again and it is working now. There must have been something else wrong when I could not get it to work the first time.

Done.